script/test

```diff
@@ -347,6 +347,7 @@ do_build_otbr_docker()
         "-DOT_UPTIME=ON"
         "-DOTBR_DNS_UPSTREAM_QUERY=ON"
         "-DOTBR_DUA_ROUTING=ON"
+        "-DOTBR_DHCP6_PD=ON"
```

Comment:
the flag should also be turned on in script/_otbr

Reply:
ACK, this will be a separate PR in ot-br-posix.